try{
	可能出现异常的语句
}catch(Exception e){
	System.out.println(e);
}finally{
	不管有没有出现异常，都会执行的代码
}
----------------------------------------------
throws		抛出异常
throw		直接抛出异常类的实例化对象

try{
	throw new Exception("实例化异常对象");
} catch(exception e){
	System.out.println(e);
}
---------------------------------------------------
自定义异常

class MyException extends Exception(String msg){
	public MyException(String msg){
		super(msg);
	}
}

主类{
	public static void main(String[] args){
		try{
			throw new MyException("自定义异常");
		} catch(MyException e){
			System.out.println(e);
		}
	}
}